bson/src/test/unit/org/bson/codecs/MapCodecV2Specification.groovy

```diff
@@ -64,11 +64,11 @@ import static org.bson.codecs.configuration.CodecRegistries.fromCodecs
 import static org.bson.codecs.configuration.CodecRegistries.fromProviders
 import static org.bson.codecs.configuration.CodecRegistries.fromRegistries
 
-class MapCodecSpecification extends Specification {
```

Comment:
We still have `IterableCodecSpecification`, but we are getting rid of `MapCodecSpecification`. Shouldn't the latter also stay? It seems reasonable to delete `IterableCodecSpecification`/`MapCodecSpecification` together with `IterableCodec`/`MapCodec`, but not before the latter are deleted.

Reply:
Done: reverted delete